package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native",
-  "version": "0.0.0-master",
+  "version": "0.22.0",
```

Comment:
This was a remnant of testing in the middle of the night and me being tired. Will fix. :)